snuba/datasets/storage.py

```diff
@@ -39,8 +39,8 @@ def __init__(self, storage_set_key: StorageSetKey, schema: Schema):
     def get_storage_set_key(self) -> StorageSetKey:
         return self.__storage_set_key
 
-    def get_cluster(self) -> ClickhouseCluster:
-        return get_cluster(self.__storage_set_key)
+    def get_cluster(self, partition_id: Optional[int] = None) -> ClickhouseCluster:
```

Comment:
While theoretically speaking this works, from a logical perspective it is not useful since we are not partitioning storages. What we are partitioning is storage sets in Snuba. So a storage would never have any idea about partitioning. So I would suggest we remove it from here.

Reply:
Sure, ok that makes sense